src/forms/EditDocumentForm.js

```diff
@@ -17,98 +16,107 @@ const EditDocumentForm = ({
   onNameChange,
   onDescriptionChange,
   onVersionStatusChange,
+  handleSubmit,
+  errors,
+  submitButtons,
 }) => {
-  const versionStatusList = Object.keys(versionState);
-  const dropdownClass = classes('Dropdown');
-  return (
-    <form className="pt-16">
-      <GridContainer>
-        <label
-          className="Form--Label row-1 cell-3 text-right Form--Label-required"
-          htmlFor="file_name"
+  const [newFileType, setFileType] = useState();
+  const [newFileName, setFileName] = useState();
+  const [newFileDescription, setFileDescription] = useState();
+  const [onUploading, setUploading] = useState(false);
+  const options = [];
+  Object.keys(versionState).forEach(function(e) {
```

Comment:
We typically opt for immutable variables when possible. Even though the `options` are declared const, it's still mutable because it's an `Array` and then immediately mutated in the `forEach`.
Instead, consider using a `map`, as is more typical in our code base:
```
const options = Object.keys(versionState).map(state => ({key: state, ...}))
```